listchans command fails with error

```
[11:03:39] [INFO] [snoonet:rory] <rory> listchans
[11:03:39] [ERROR] Error in hook admin_bot:listchans
Traceback (most recent call last):
  File "/home/rory/repos/CloudBot/cloudbot/plugin.py", line 376, in _execute_hook
    out = yield from self.bot.loop.run_in_executor(None, self._execute_hook_threaded, hook, event)
  File "/usr/lib/python3.4/asyncio/futures.py", line 386, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 287, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 275, in result
    raise self._exception
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/rory/repos/CloudBot/cloudbot/plugin.py", line 340, in _execute_hook_threaded
    return hook.function(*parameters)
  File "/home/rory/repos/CloudBot/plugins/admin_bot.py", line 390, in listchans
    lines = formatting.chunk_str("I am currently in: {}".format(chans))
NameError: name 'formatting' is not defined
```

--- a/plugins/admin_bot.py
+++ b/plugins/admin_bot.py
@@ -2,6 +2,7 @@ import asyncio
 import re
 
 from cloudbot import hook
+from cloudbot.util import formatting
 
 logchannel = ""
 

